suzieq/shared/utils.py

```diff
@@ -460,11 +460,26 @@ def convert_macaddr_format_to_colon(macaddr: str) -> str:
 
     """
     if isinstance(macaddr, str):
-        if re.match(r'[0-9a-zA-Z]{4}.[0-9a-zA-Z]{4}.[0-9a-zA-Z]{4}', macaddr):
+        macaddr = macaddr.lower()
```

Comment:
If the mac address is already in the colon format it will return `00:00:00:00:00:00`. Can you add a check to verify that? something like
```
if re.match(r'([0-9a-f]{2}\:){3}[0-9a-f]{2}\:?', macaddr):
    return macaddr
```